### ISSUE TITLE
Plugin throws exception when checking for inspection changes

### ISSUE THREAD

**Issue by zzzz-best** [frimtec/teamcity-inspection-notification-plugin#182]
I have installed the plugin on TeamCity Enterprise 2022.04.1 (build 108575), The test email works but the emails on inspections change doesn't.

Upoon looking in the server logs i see this message:

[2023-09-06 11:47:36,536]   WARN - ildServer.util.EventDispatcher - Error in event handler: Error calling method BuildServerListener.buildFinished for listener com.github.frimtec.teamcity.plugin.inspectionnotification.InspectionNotificationBuildListener: org.springframework.jdbc.BadSqlGrammarException: ConnectionCallback; bad SQL grammar []; nested exception is com.microsoft.sqlserver.jdbc.SQLServerException: Invalid object name 'INSPECTION_DIFF'. (enable debug to see stacktrace)

It appears something to do with the SQL query to get the inspection_diff table

**Comment by frimtec** [comment]
Hi @zzzz-best 
Thx for opening this issue.

The SQL query is stable since years, therefore I'm a bit surprised. 
In our company we use Oracle 19c as the database. The plugins integration tests use hsqldb. 
Maybe, there is an incompatibility with your database.
What database (including version) do you use?
Can you execute the following query against your database manually, replacing ```<build_id>``` with an existing build id?
```
select distinct IDA.SEVERITY as SEVERITY,
                IIN.INSPECTION_NAME as INSPECTION_NAME,
                IDA.FILE_NAME as FILE_NAME,
                IRE.LINE as LINE
           from INSPECTION_DIFF IDI
                  join INSPECTION_DATA IDA on (IDA.HASH = IDI.HASH)
                  join INSPECTION_INFO IIN on (IIN.ID = IDA.INSPECTION_ID)
                  join INSPECTION_RESULTS IRE on (IRE.HASH = IDI.HASH)
          where IDI.BUILD_ID = <build_id>;
```

**Comment by zzzz-best** [comment]
Hi frimtec, thanks for the quick response, appreciate it!

Unfortunately I cannot access the server machine and perform the query. I am guessing there is not an alternate way of performing that query from within the configuration of Teamcity?

The database type is MSSQL Express, I am not quite sure where I can find the version without access to the server machine (perhaps its reported in one of the log files?)

I wonder if perhaps if this may be anything like case sensitivity to the table names in this particular database?

**Comment by frimtec** [comment]
Can you replace the plugin in teamcity with a patched un-official version?
If yes, I can prepare a test version where the query uses lowercase table and column names.

**Comment by zzzz-best** [comment]
Yes thank you that would be great! I am able to install plugins from zip.

**Comment by frimtec** [comment]
You can find a patched version here (Version 1.1.3-beta-1): https://github.com/frimtec/teamcity-inspection-notification-plugin/releases/download/1.1.3/inspection-violation-notification.zip

**Comment by zzzz-best** [comment]
Hi frimtec really appreciate it, it is working now! Thank you

[2023-09-07 08:51:39,009]   INFO - tbrains.buildServer.ACTIVITIES - InspectionNotificationPlugin: Build: <PROJECT>; New violations mail sent to: aaa@aaa.com.

[2023-09-07 08:47:33,923]   INFO - tbrains.buildServer.ACTIVITIES - InspectionNotificationPlugin: Build: <PROJECT>; No new violations in build.

![image](https://github.com/frimtec/teamcity-inspection-notification-plugin/assets/144198952/a332270b-0d99-41e8-b787-abd11347f966)

**Comment by frimtec** [comment]
Fixed with release 1.1.3.
